analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/TokenTypeAnalyzerTest.Classifier.cs

```diff
@@ -805,6 +805,24 @@ public class Test: IFormattable
             }
             """, allowSemanticModel);
 
+    [DataTestMethod]
+    [DataRow("[t:Int32]?", false)]
+    [DataRow("[u:System].Int32?", true)]
+    [DataRow("[t:IDisposable]?", false)]
+    [DataRow("[t:IDisposable]?[]", false)]
+    [DataRow("[t:IDisposable]?[]?", false)]
```

Comment:
Could you please add also a test with `int?`? In most cases we classify `int` as keyword.